extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -447,16 +473,42 @@ object PrivilegesBuilder {
         val table = getPlanField[TableIdentifier]("table")
         inputObjs += tablePrivileges(table)
 
+      case "ShowTableProperties" =>
+        val resolvedTable = getPlanField[Any]("table")
+        val identifier = getFieldVal[AnyRef](resolvedTable, "identifier")
+        val namespace = invoke(identifier, "namespace").asInstanceOf[Array[String]]
+        val table = invoke(identifier, "name").asInstanceOf[String]
+        inputObjs += PrivilegeObject(
+          TABLE_OR_VIEW,
+          PrivilegeObjectActionType.OTHER,
+          quote(namespace),
+          table,
+          Nil)
+
+      case "ShowCreateTable" =>
+        val resolvedTable = getPlanField[Any]("child")
+        val identifier = getFieldVal[AnyRef](resolvedTable, "identifier")
+        val namespace = invoke(identifier, "namespace").asInstanceOf[Array[String]]
+        val table = invoke(identifier, "name").asInstanceOf[String]
+        inputObjs += PrivilegeObject(
+          TABLE_OR_VIEW,
+          PrivilegeObjectActionType.OTHER,
+          quote(namespace),
+          table,
+          Nil)
+
       case "ShowFunctionsCommand" =>
-        getPlanField[Option[String]]("db").foreach { db =>
-          inputObjs += databasePrivileges(db)
-        }
```

Comment:
why we need this change

Reply:
using filtered command instead of this

Reply:
I see it is not for compatibility but for clean up